docs/pages/blog/introducing-pigment-css.md

```diff
@@ -78,19 +85,50 @@ Check out [How Pigment CSS works](https://github.com/mui/pigment-css/blob/maste
+| :--------------------------- | -------------------------------------------------------------------------------------------------------------: | -----------------------------------------------------------------------------------------------------------------: | -----: |
+| HTML Size                    |                                                                                                         15.8kB |                                                                                                             15.0kB |    -5% |
+| First Load JS                |                                                                                                          131kB |                                                                                                              104kB |   -20% |
+| Total blocking time          | [280 ms](https://pagespeed.web.dev/analysis/https-emotion-demo-vercel-app/uxfpkvbp31?hl=fr&form_factor=mobile) | [210 ms](https://pagespeed.web.dev/analysis/https-pigment-css-demo-vercel-app/azcw1qxkec?hl=fr&form_factor=mobile) |   -25% |
+
+:::info
+**Reproduction**. Because you should only trust numbers that you can reproduce, we've shared below how we found these numbers.
+
+- Emotion
+  - Live: https://emotion-demo.vercel.app/?metric=1
+  - Source: https://github.com/brijeshb42/emotion-demo
+- Pigment CSS
+  - Live: https://pigment-css-demo.vercel.app/?metric=1
+  - Source: https://github.com/brijeshb42/pigment-css-landing-page-demo
+
+:::
+
+### Better runtime performance
 
-When comparing the same Material UI app built with Next.js and either Emotion or Pigment CSS, we've observed the following page load performance gains:
+When comparing the same Material UI app built with Next.js and either Emotion or Pigment CSS, we've observed the following runtime performance gains using the same code, but powered by different style engines:
 
-| Metrics                | Emotion | Pigment CSS | Reduction |
-| :--------------------- | ------: | ----------: | --------: |
-| First load JavaScript  |   131kB |       104kB |       20% |
-| First Contentful Paint |   503ms |       455ms |        9% |
-| Time To First Byte     | 447.5ms |     381.5ms |       15% |
-| Total Page HTML        |  15.9kB |      14.7kB |      7.5% |
+<!-- After page load, runtime performance is also improved with Pigment CSS. -->
+
+| Metrics                                 | Emotion | Pigment CSS | Change |
+| :-------------------------------------- | ------: | ----------: | -----: |
+| Create and mount a new button           |  17.3ms |      10.1ms |   -42% |
+| Change a variant on a mounted component |  14.0ms |      9.13ms |   -34% |
+| Change a value inside a CSS prop        |  13.6ms |      8.63ms |   -37% |
```

Comment:
@brijeshb42 I think we should have a baseline for each test, using CSS Modules or Tailwind CSS if needed to have it.

The other tests, I could add them, there was no real performance progress. Which seems to match with https://pigment-css-bench.vercel.app/, it's not faster. Something might be off.